### ISSUE TITLE
[Android] Is it legal to use telegram stickers for my app ? @simonzhexu

### ISSUE THREAD

**Issue by AnasCode** [WhatsApp/stickers#297]
I see a lot of people using telegram stickers on their apps , and some of the developers say they have obtained the permission from Telegram , does anyone have an idea about that ? i want to feature some packs from telegrams too and i'm wondering if it's legal ! 
PS : i know the issue might sound like it's off topic but i wish you can answer because i've already made an app and use some packs from Telegram and yet haven't submit it because i don't want to do something illegal !

**Comment by oas** [comment]
You need to contact telegram at stickers@telegram.org.

**Comment by AnasCode** [comment]
@oas Have you contact them ? do they answer ? i've sent them an email two hours ago and still didn't hear anything from them .

**Comment by oas** [comment]
Yes, I contacted them a few weeks ago. Be patient, they're doing you a favor.

**Comment by AnasCode** [comment]
Did they let you use them ? the stickers i want to use are not offical stickers from Telegram , i mean you need first to add them to Telegram and then you can use them , so once they're on Telegram and Telegram gives me permission i can use those too right ? ( an example of what i'm talking about is this website : http://tlgrm.eu/ ) 
PS : how long did it take to get permission ? 
Thanks by the way !

**Comment by LouisWayne** [comment]
@AnasCode What did they say?

**Comment by Josepdal** [comment]
@LouisWayne 
https://telegra.ph/Sharing-Telegram-Stickers-to-WhatsApp-10-29

**Comment by karkouri990** [comment]
Look at this site https://tgchannels.net
